.azure-pipelines/gpu-tests.yml

```diff
@@ -50,7 +50,7 @@ jobs:
 
     - bash: |
         python -c "fname = 'requirements/extra.txt' ; lines = [line for line in open(fname).readlines() if 'horovod' not in line] ; open(fname, 'w').writelines(lines)"
-        pip install fairscale==0.4.0
+        pip install fairscale==0.4.5
```

Comment:
There's a failing test:

tests/strategies/test_ddp_fully_sharded_with_full_state_dict.py::test_fully_sharded_strategy_checkpoint

Reply:
Thanks @carmocca 
It caused by a fairscale bug fix  "[Fixed a corner case of FSDP init order and losing one of the flags](https://github.com/facebookresearch/fairscale/commit/7c2c3e0046eb3a1b1c98fd24f61fb09f90f6817f)" in version 0.4.4
Our tests needs to be updated, I will update them later

Reply:
The ci gpu is broken, and I couldn't reproduce this on my machine, so will have to wait

Reply:
@four4fish The CI should be back now. Would you mind having a look?